content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -92,6 +76,10 @@ EndpointSlices support three address types:
 * IPv6
 * FQDN (Fully Qualified Domain Name)
 
+EndpointSlices represent a particular IP address family. If you have a Service that
+is available via IPv4 and also via IPv6, there will be at least two EndpointSlices
+(one for IPv4, and one for IPv6).
```

Comment:
Is this right?

Reply:
```suggestion
Each `EndpointSlice` object represents a specific IP address type. If you have
a Service that is available via IPv4 and IPv6, there will be at least two
`EndpointSlice` objects (one for IPv4, and one for IPv6).
```

This also should probably clarify what "at least" means - if there's more than one IPv4 address used, will there be additional EndpointSlice objects?

Reply:
>  if there's more than one IPv4 address used, will there be additional EndpointSlice objects

There might or might not be.  Maybe this wording:
```markdown
If you have a `Service` that is available via IPv4 and IPv6, there will be at least two
`EndpointSlice` objects (one or more `EndpointSlice` for IPv4, and one or more
`EndpointSlice` for IPv6).
```

?

(aside: the neither-option-feels-right pluralization is a key reason why I don't like using backticks for the names of API kinds)